### PR TITLE
Small fix for skill gems

### PR DESCRIPTION
**Pull request by impbk2002**
Corpse Explosion have 0 crit chance as default, and cannot add attack damage or spell damage modifier except from support gem

-fixed
 infused channeling alt quality having duplicate effect
 sand and stone alt quality (armor -> armour )

### CODE REVIEW

File: Data/Skills/act_dex.lua [hunk 1]
```diff
@@ -1811,6 +1811,17 @@ skills["CorpseEruption"] = {
 	},
 	baseMods = {
 		skill("explodeCorpse", true, { type = "SkillPart", skillPart = 2 }),
+		skill("CritChance",0,{ type = "SkillPart", skillPart = 2 }),
+		skill("PhysicalMin", 0,{ type = "SkillPart", skillPart = 2 }),
+		skill("PhysicalMax",0,{ type = "SkillPart", skillPart = 2 }),
+		skill("LightningMin",0,{ type = "SkillPart", skillPart = 2 }),
+		skill("LightningMax",0,{ type = "SkillPart", skillPart = 2 }),
+		skill("ColdMin",0,{ type = "SkillPart", skillPart = 2 }),
```
Comment (ppoelzl) [inline Data/Skills/act_dex.lua:1819]: Add whitespace after commas.

File: Data/Skills/sup_int.lua [hunk 2]
```diff
@@ -1950,7 +1950,7 @@ skills["SupportEnergyShieldLeech"] = {
 	description = "Supports any skill that hits enemies, causing those hits to leech energy shield based on damage dealt.",
 	color = 3,
 	support = true,
-	requireSkillTypes = { SkillType.Hit, SkillType.Attack, },
+	requireSkillTypes = { SkillType.Hit, },
```
Comment (ppoelzl) [inline Data/Skills/sup_int.lua:1953]: Remove hanging commas in single-line tables or functions.

**Review by ppoelzl — COMMENTED** [review]
Right at the top of the files you edited, there's this notice:
`-- This file is automatically generated, do not edit!`
Instead, edit the export scripts (the corresponding .txt files), run the exporter, and include the changes to both in your PR.
Otherwise, your changes will get overwritten the next time we run the export scripts for a new release.

All of this is reflected in our [CONTRIBUTING](https://github.com/PathOfBuildingCommunity/PathOfBuilding/blob/master/CONTRIBUTING.md) file. I very much urge you to fully read it. If there is anything in there that's unclear to you, please let us know.

The formatting of this PR as well as your other PRs is very inconsistent, both in itself and with the surrounding code. In their current state, we are not able to accept your contributions without extensive corrections. If there is anything code style-wise that's unclear to you, please also let us know.

> also other small errors can be fixed by this PR

Please fully state all changes in your PRs, so reviewers don't have to guess your intentions.
Also please don't conflate non-overlapping features or fixes into a single PR.
For a collection of small bug fixes like here, it's okay to blend them in a single PR if and only if they are in separate, comprehensibly-named commits.

I haven't looked into your other PRs that much, but they give the impression that much of what I wrote here also applies to them.
Until you fixed at least all formatting issues, I will not review them.
For most of them, I suggest either opening a new PR or force-pushing over the existing one to keep the git history more readable.

Please don't let this lengthy text discourage you from contributing! We all have to start somewhere and it's okay to make mistakes. The fixes themselves that you made look very promising. It's just the style you have to look over again.

If you have any questions, please let me know.

**Comment by impbk2002** [comment]
> Right at the top of the files you edited, there's this notice:
> `-- This file is automatically generated, do not edit!`
> Instead, edit the export scripts (the corresponding .txt files), run the exporter, and include the changes to both in your PR.
> Otherwise, your changes will get overwritten the next time we run the export scripts for a new release.
> 
> All of this is reflected in our [CONTRIBUTING](https://github.com/PathOfBuildingCommunity/PathOfBuilding/blob/master/CONTRIBUTING.md) file. I very much urge you to fully read it. If there is anything in there that's unclear to you, please let us know.
> 
> The formatting of this PR as well as your other PRs is very inconsistent, both in itself and with the surrounding code. In their current state, we are not able to accept your contributions without extensive corrections. If there is anything code style-wise that's unclear to you, please also let us know.
> 
> > also other small errors can be fixed by this PR
> 
> Please fully state all changes in your PRs, so reviewers don't have to guess your intentions.
> Also please don't conflate non-overlapping features or fixes into a single PR.
> For a collection of small bug fixes like here, it's okay to blend them in a single PR if and only if they are in separate, comprehensibly-named commits.
> 
> I haven't looked into your other PRs that much, but they give the impression that much of what I wrote here also applies to them.
> Until you fixed at least all formatting issues, I will not review them.
> For most of them, I suggest either opening a new PR or force-pushing over the existing one to keep the git history more readable.
> 
> Please don't let this lengthy text discourage you from contributing! We all have to start somewhere and it's okay to make mistakes. The fixes themselves that you made look very promising. It's just the style you have to look over again.
> 
> If you have any questions, please let me know.

It's my first time doing PR or github and Lua, so I apologize for any inconvenient you have from my PRs. 
I accept the inconsistency of format in my PRs against original POB. Before editing my PRs, is there anything to consider in making consistency, except whitespace, brackets, and commas?

**Comment by impbk2002** [comment]
> Right at the top of the files you edited, there's this notice:
> `-- This file is automatically generated, do not edit!`
> Instead, edit the export scripts (the corresponding .txt files), run the exporter, and include the changes to both in your PR.
> Otherwise, your changes will get overwritten the next time we run the export scripts for a new release.
> 
> All of this is reflected in our [CONTRIBUTING](https://github.com/PathOfBuildingCommunity/PathOfBuilding/blob/master/CONTRIBUTING.md) file. I very much urge you to fully read it. If there is anything in there that's unclear to you, please let us know.
> 
> The formatting of this PR as well as your other PRs is very inconsistent, both in itself and with the surrounding code. In their current state, we are not able to accept your contributions without extensive corrections. If there is anything code style-wise that's unclear to you, please also let us know.
> 
> > also other small errors can be fixed by this PR
> 
> Please fully state all changes in your PRs, so reviewers don't have to guess your intentions.
> Also please don't conflate non-overlapping features or fixes into a single PR.
> For a collection of small bug fixes like here, it's okay to blend them in a single PR if and only if they are in separate, comprehensibly-named commits.
> 
> I haven't looked into your other PRs that much, but they give the impression that much of what I wrote here also applies to them.
> Until you fixed at least all formatting issues, I will not review them.
> For most of them, I suggest either opening a new PR or force-pushing over the existing one to keep the git history more readable.
> 
> Please don't let this lengthy text discourage you from contributing! We all have to start somewhere and it's okay to make mistakes. The fixes themselves that you made look very promising. It's just the style you have to look over again.
> 
> If you have any questions, please let me know.

**Comment by ppoelzl** [comment]
This PR looks fine, logic wise. I'll comment on the others in their respective PRs.

File: Data/Skills/act_dex.lua [hunk 3]
```diff
@@ -1811,6 +1811,17 @@ skills["CorpseEruption"] = {
 	},
 	baseMods = {
 		skill("explodeCorpse", true, { type = "SkillPart", skillPart = 2 }),
+		skill("CritChance", 0, { type = "SkillPart", skillPart = 2 }),
+		skill("PhysicalMin", 0,{ type = "SkillPart", skillPart = 2 }),
```
Comment (ppoelzl) [inline Data/Skills/act_dex.lua:1815]:
```suggestion
		skill("PhysicalMin", 0, { type = "SkillPart", skillPart = 2 }),
```

File: Data/Skills/act_dex.lua [hunk 4]
```diff
@@ -2313,6 +2324,17 @@ skills["DetonateDead"] = {
 	baseMods = {
 		skill("radius", 22),
 		skill("explodeCorpse", true, { type = "SkillPart", skillPart = 2 }),
+		skill("CritChance", 0, { type = "SkillPart", skillPart = 2 }),
+		skill("PhysicalMin", 0,{ type = "SkillPart", skillPart = 2 }),
```
Comment (ppoelzl) [inline Data/Skills/act_dex.lua:2328]:
```suggestion
		skill("PhysicalMin", 0, { type = "SkillPart", skillPart = 2 }),
```

File: Data/Skills/act_dex.lua [hunk 5]
```diff
@@ -2403,6 +2425,17 @@ skills["VaalDetonateDead"] = {
 	baseMods = {
 		skill("explodeCorpse", true, { type = "SkillPart", skillPart = 2 }),
 		skill("radius", 24),
+		skill("CritChance", 0, { type = "SkillPart", skillPart = 2 }),
+		skill("PhysicalMin", 0,{ type = "SkillPart", skillPart = 2 }),
```
Comment (ppoelzl) [inline Data/Skills/act_dex.lua:2429]:
```suggestion
		skill("PhysicalMin", 0, { type = "SkillPart", skillPart = 2 }),
```

File: Data/Skills/act_dex.lua [hunk 6]
```diff
@@ -8244,6 +8277,17 @@ skills["VolatileDead"] = {
 	baseMods = {
 		skill("explodeCorpse", true, { type = "SkillPart", skillPart = 2 }),
 		skill("radius", 15),
+		skill("CritChance", 0, { type = "SkillPart", skillPart = 2 }),
+		skill("PhysicalMin", 0,{ type = "SkillPart", skillPart = 2 }),
```
Comment (ppoelzl) [inline Data/Skills/act_dex.lua:8281]:
```suggestion
		skill("PhysicalMin", 0, { type = "SkillPart", skillPart = 2 }),
```

File: Data/Skills/act_int.lua [hunk 7]
```diff
@@ -1102,6 +1102,17 @@ skills["CorpseWarp"] = {
 	baseMods = {
 		skill("explodeCorpse", true, { type = "SkillPart", skillPart = 2 }),
 		skill("radius", 10),
+		skill("CritChance", 0, { type = "SkillPart", skillPart = 2 }),
+		skill("PhysicalMin", 0,{ type = "SkillPart", skillPart = 2 }),
```
Comment (ppoelzl) [inline Data/Skills/act_int.lua:1106]:
```suggestion
		skill("PhysicalMin", 0, { type = "SkillPart", skillPart = 2 }),
```

File: Data/Skills/act_dex.lua (hunk 1 above)
Comment (ppoelzl) [reply Data/Skills/act_dex.lua]: There is still one instance of incorrect formatting present in all all corpse skill related files.

File: Data/Skills/sup_int.lua (hunk 2 above)
Comment (ppoelzl) [reply Data/Skills/sup_int.lua:1953]: You have not addressed this comment yet. Do you need any further instructions?

**Review by ppoelzl — COMMENTED** [review]
Do not append newlines to the export scripts or export data. Please start over with this PR and separate it into individual PRs or commits for each fix (corpse explosions, infused channeling, flesh and stone (not sand and stone!))

**Pull request closed without merging**